### PR TITLE
Blog

### PR DESCRIPTION
**Pull request by mrosso10**
# Summary
* Implemented blog app from Rails tutorial

# Test plan
1. Go to the home page
2. Click on the 'New article' link
3. Login with username 'dhh' and password 'secret'
4. Create a new article
5. Go to /articles
6. Click on the 'Edit' link and update the created article
7. Create a new comment for the article
8. Go back to /articles and destroy the article

# Trello Card
https://trello.com/c/6f7onHIE/22-crear-blog-base

### CODE REVIEW

File: app/assets/javascripts/articles.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (Dinuuu) [inline app/assets/javascripts/articles.coffee]: Try to delete the files that you are not using to keep the proyect as clean as posible

File: app/assets/javascripts/comments.coffee [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (Dinuuu) [inline app/assets/javascripts/comments.coffee]: idem

File: app/assets/javascripts/welcome.coffee [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (Dinuuu) [inline app/assets/javascripts/welcome.coffee]: idem

File: app/assets/stylesheets/articles.scss [hunk 4]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the articles controller here.
```
Comment (Dinuuu) [inline app/assets/stylesheets/articles.scss]: idem

File: app/assets/stylesheets/comments.scss [hunk 5]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Comments controller here.
```
Comment (Dinuuu) [inline app/assets/stylesheets/comments.scss]: idem

File: app/assets/stylesheets/welcome.scss [hunk 6]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the welcome controller here.
```
Comment (Dinuuu) [inline app/assets/stylesheets/welcome.scss]: idem

File: app/helpers/application_helper.rb [hunk 7]
```diff
@@ -0,0 +1,2 @@
+module ApplicationHelper
```
Comment (Dinuuu) [inline app/helpers/application_helper.rb]: idem

File: app/helpers/articles_helper.rb [hunk 8]
```diff
@@ -0,0 +1,2 @@
+module ArticlesHelper
```
Comment (Dinuuu) [inline app/helpers/articles_helper.rb]: idem

File: app/helpers/comments_helper.rb [hunk 9]
```diff
@@ -0,0 +1,2 @@
+module CommentsHelper
```
Comment (Dinuuu) [inline app/helpers/comments_helper.rb]: idem

File: app/helpers/welcome_helper.rb [hunk 10]
```diff
@@ -0,0 +1,2 @@
+module WelcomeHelper
```
Comment (Dinuuu) [inline app/helpers/welcome_helper.rb]: idem

File: config/initializers/backtrace_silencers.rb [hunk 11]
```diff
@@ -0,0 +1,7 @@
+# Be sure to restart your server when you modify this file.
```
Comment (Dinuuu) [inline config/initializers/backtrace_silencers.rb]: idem

File: config/initializers/inflections.rb [hunk 12]
```diff
@@ -0,0 +1,16 @@
+# Be sure to restart your server when you modify this file.
```
Comment (Dinuuu) [inline config/initializers/inflections.rb]: idem

File: config/initializers/mime_types.rb [hunk 13]
```diff
@@ -0,0 +1,4 @@
+# Be sure to restart your server when you modify this file.
```
Comment (Dinuuu) [inline config/initializers/mime_types.rb]: idem

File: db/seeds.rb [hunk 14]
```diff
@@ -0,0 +1,7 @@
+# This file should contain all the record creation needed to seed the database with its default values.
```
Comment (Dinuuu) [inline db/seeds.rb]: idem

File: test/controllers/articles_controller_test.rb [hunk 15]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```
Comment (Dinuuu) [inline test/controllers/articles_controller_test.rb]: idem

File: test/controllers/comments_controller_test.rb [hunk 16]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```
Comment (Dinuuu) [inline test/controllers/comments_controller_test.rb]: idem

File: test/models/article_test.rb [hunk 17]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
```
Comment (Dinuuu) [inline test/models/article_test.rb]: idem

File: test/models/comment_test.rb [hunk 18]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
```
Comment (Dinuuu) [inline test/models/comment_test.rb]: idem

File: app/controllers/articles_controller.rb [hunk 19]
```diff
@@ -0,0 +1,47 @@
...
+  
+  def create
+    @article = Article.new(article_params)
+   
+    if @article.save
+      redirect_to @article
+    else
+      render 'new'
+    end
+    # render plain: params[:article].inspect
```
Comment (Dinuuu) [inline app/controllers/articles_controller.rb]: For the same reasons we try to keep the code clean of comments (unless they are to explain something)

File: config/database.yml [hunk 20]
```diff
@@ -0,0 +1,44 @@
+# SQLite version 3.x
+#   gem install sqlite3
+#
+#   Ensure the SQLite 3 gem is defined in your Gemfile
+#   gem 'sqlite3'
+#
+default: &default
```
Comment (Dinuuu) [inline config/database.yml]: This default is not necessary since you are not using it, so you can erase it

File: config/database.yml [hunk 21]
```diff
@@ -0,0 +1,44 @@
...
+#   database: db/development.sqlite3
+
+# Warning: The database defined as "test" will be erased and
+# re-generated from your development database when you run "rake".
+# Do not set this db to the same as development or production.
+# test:
+#   <<: *default
+#   database: db/test.sqlite3
+
+production:
```
Comment (Dinuuu) [inline config/database.yml]: idem default

**Comment by Dinuuu** [comment]
did you use Renv and Ruby-Build to install Ruby ? I can't see the `.ruby_version` file

**Comment by Dinuuu** [comment]
Nice Commit separation ! You isolate the requirements really well.

**Comment by mrosso10** [comment]
Yes, I used Rbenv and Ruby-Build but I used the default config file to set the Ruby version. I now added the .ruby_version file

**Pull request closed without merging**